nanshe_workflow/par.py

```diff
@@ -650,9 +650,12 @@ def __iter__(self):
                 def __len__(self):
                     return(len(self.data_blocks_dict_sample))
 
-            calculate_block = lambda db, dbds, kw: zarr.array(calculate(
-                db[...], dbds[...], *new_args, **kw
-            ))
+            def calculate_block(db, dbds, kw):
+                with dask.set_options(get=dask.get):
+                    return zarr.array(calculate(
+                        numpy.asarray(db[...]), numpy.array(dbds[...]), *new_args, **kw
```

Comment:
Should have been using `numpy.asarray` in the second call, but used `numpy.array` instead. Fixing in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/186 ).